kubejs/server_scripts/enigmatica/kubejs/base/recipes/remove.js

```diff
@@ -83,6 +91,13 @@ events.listen('recipes', function (event) {
         mod: 'buildersaddition',
         type: 'minecraft:crafting_shaped'
     });
+
+    event.remove({
+        output: '/thermal:parts/\\w+_vertical_slab/',
+        mod: 'buildersaddition',
+        type: 'minecraft:crafting_shaped'
+    });
+

```

Comment:
What's going on here? :)

Reply:
Oh shoot, sorry. I  was copying that little guy to edit and use for the gear removals. thought I backed out that change though